### PR TITLE
Add testing component

### PR DESCRIPTION
**Pull request by chapulina**
Component added in https://github.com/ignitionrobotics/ign-common/pull/314

Test builds:

* Focal [Build Status](https://build.osrfoundation.org/job/ign-common5-debbuilder/328/), new [Build Status](https://build.osrfoundation.org/job/ign-common5-debbuilder/330/), new [Build Status](https://build.osrfoundation.org/job/ign-common5-debbuilder/331/)
* Jammy [Build Status](https://build.osrfoundation.org/job/ign-common5-debbuilder/329/)

### CODE REVIEW

**Comment by scpeters** [comment]
I don't see any files added to `focal/debian`, which might explain why the most recent build is unstable

**Comment by chapulina** [comment]
I had forgotten to add the files 🤦🏽‍♀️

File: ubuntu/debian/control [hunk 1]
```diff
@@ -250,3 +251,34 @@ Description: Collection of useful code used by robotics apps - Profiler dev file
...
+ libraries designed to rapidly develop robot applications. A collection of
+ useful classes and functions for handling many command tasks. This includes
+ parsing 3D mesh files, managing console output, and using PID controllers.
+ .
+ Testing component of the library
+
+Package: libignition-common5-testing-dev
+Architecture: any
+Section: libdevel
+Depends: libgdal-dev,
```
Comment (scpeters) [inline ubuntu/debian/control]: I think it shouldn't depend on `libgdal-dev`

Comment (mjcarroll) [reply ubuntu/debian/control]: Agree, that is strange.

Comment (chapulina) [reply ubuntu/debian/control]: Bad copy-paste, fixed in [d9db5d8](https://github.com/ignition-release/ign-common5-release/pull/10/commits/d9db5d8b970c6f80e6de37b8626b04c3069785c2)

**Comment by chapulina** [comment]
I'm merging this to fix downstream CI that's failing with 

```
dpkg: error processing archive /tmp/apt-dpkg-install-vxCq7C/023-libignition-common5-core-dev_4.999.999+nightly+git20220420+1r1b1ad37b31be684b3679d8b92cca957addeaab2f-1~focal_amd64.deb (--unpack):
 trying to overwrite '/usr/include/ignition/common5/ignition/common/testing.hh', which is also in package libignition-common5-testing-dev:amd64 
```

**Pull request merged**